runeq/resources/project.py

```diff
@@ -97,7 +403,7 @@ def _item_class(self) -> Type[ItemBase]:
 
 
 def get_project(
-    project_id: str, client: Optional[GraphClient] = None
+    id: str, client: Optional[GraphClient] = None
```

Comment:
I think this should remain as `project_id`, since similar methods like `get_patient` have args for `patient_id` not `id`. What motivated the change here?

Reply:
That sounds good to me!